src/app/chart-list/chart-list.component.html

```diff
@@ -1,4 +1,8 @@
 <section class="chart-list">
-  <h3 *ngIf="!charts || !charts.length" class="chart-list__empty">There are no charts</h3>
+  <div *ngIf="apiNotReady" class="chart-list__empty">
+    <app-loader [loading]="apiNotReady"></app-loader>
+    <h3>The Dashboard API is still starting up. This may take a few minutes...</h3>
```

Comment:
I'd complete this message with: "This page will refresh automatically when the dashboard is ready to use."

Reply:
The loading icon reflects that, I think. I would not make this more verbose.

Reply:
<img width="1315" alt="screen shot 2017-11-16 at 10 52 53" src="https://user-images.githubusercontent.com/1544861/32887595-54644474-cabc-11e7-8769-82d028a73503.png">

Reply:
I didn't notice the loader sorry. I agree it's fine :)

Reply:
NP, my bad for not including the screenshot!